src/lucky/tags/input_helpers.cr

```diff
@@ -30,13 +30,33 @@ module Lucky::InputHelpers
 
   generate_helpful_error_for textarea
 
+  # Returns a textarea field.
+  #
+  # ```
+  # textarea(attribute)
+  # # => <textarea id="param_key_attribute_name" name="param_key:attribute_name"></textarea>
+  # ```
   def textarea(field : Avram::PermittedAttribute, **html_options)
     textarea field.param.to_s, merge_options(html_options, {
       "id"   => input_id(field),
       "name" => input_name(field),
     })
```

Comment:
In order to reduce the similar code in these, I think this should call the other method with empty html attrs

```crystal
textarea(field: field, attrs: [] of Symbol, **html_options)
```

Same with the other methods